new/rfcXXX-lifecycle-and-release-policies.md

```diff
@@ -0,0 +1,203 @@
+
+Chef OSS that falls under Cadence Release is under heavy development by the community and Chef Software and is released on a regular cadence. Cadence Releases help ensure that Chef Maintainers get new features and minor bug fixes out to users in a timeline fashion while not overloading the users with numerous releases and monotonically growing version numbers.
+
+Because of the high velocity of development, it is possible for many new features to be added during any given release cycle. To keep the version numbers of [Stable Releases](#stable-release) meaningful, the **MINOR** version will increment by one for each Stable Release. This means that many new features may be added between two Stable Releases, but the **MINOR** version will only increment by one.
+
+Because so much can change from release to release, maintainers will select a `MAJOR.MINOR.PATCH` build as a [Release Candidate](#release-candidate). Maintainers will follow a slightly modified [Release Communication](#release-communication) procedure to make an announcement to the "chef-dev" [Discourse](https://discourse.chef.io) category, notifying users of the impending Stable Release. The maintainers will then wait a pre-determined amount of time to allow user to perform further testing. If no critical regressions or other release blockers are identified, the Stable Release will proceed as scheduled.
+
+The following projects adhere to Cadence Release:
+
+| Chef OSS | MAJOR Release Cadence | MINOR Release Cadence | Release Candidate Delay |
+|---|---|---|---|
+| Chef Client | Yearly (April)[<sup>1</sup>](#footnotes) | Monthly (2nd Week) | 3 business days |
+| ChefDK | Yearly (April) | Monthly (4th Week)[<sup>2</sup>](#footnotes) | 3 business days |
+
+#### Ad-hoc Release
+
+[![](https://img.shields.io/badge/OSS%20Lifecycle-Ad--hoc%20Release-green.svg)](https://github.com/chef/chef-rfc/blob/master/rfcXXX-versioning-and-release-policy.md#ad-hoc-release)
+
+Chef OSS that falls under Ad-hoc Release is actively developed by the community and Chef Software but not at such a velocity as to necessitate a cadence release. Instead, software releases will be made on an as-needed basis as determined by the maintainers for that software.
+
+Unlike Cadence Release software, maintainers may decide to release multiple **PATCH** artifacts as Stable Releases (e.g. 1.2.3 and 1.2.5 may both be Stable Releases). Communication regarding these releases will still follow the standard Release Communication procedure.
+
+#### Maintenance Mode
+
+[![](https://img.shields.io/badge/OSS%20Lifecycle-Maintenance%20Mode-lightgrey.svg)](https://github.com/chef/chef-rfc/blob/master/rfcXXX-versioning-and-release-policy.md#maintenance-mode)
+
+Chef OSS that is in Maintenance Mode is no longer being actively developed but releases are still made in the case a bug fix is required. The details in terms of versioning and release processes are the same as Ad-hoc Releases.
```

Comment:
This overlaps with RFC-015 which defined the "N-1" policy and which we explicitly dropped the "bug fix" language out of it because we were not following that and it was causing user confusion (and angry users jumping around between github issues and support trying to lawyer us into doing a backport and release of a bugfix that we were clearly never going to do -- including support agreeing that the written policy suggested we should do that):

https://github.com/chef/chef-rfc/pull/140